src/package/settings/view/network/interface/Ipv4.js

```diff
@@ -56,6 +56,32 @@ Ext.define('Mfw.settings.interface.Ipv4', {
                     required: '{intf.configType === "ADDRESSED" && intf.type !== "OPENVPN" && intf.type !== "WWAN"}'
                 }
             }, {
+                xtype: 'container',
+                padding: '0 0 16 0',
+                layout: {
+                    type: 'hbox',
+                    align: 'middle'
+                },
+                hidden: true,
+                bind: {
+                    hidden: '{intf.v4ConfigType !== "DHCP"}'
+                },
+                items: [{
+                    xtype: 'component',
+                    flex: 1,
+                    bind: {
+                        html: 'Current address: <strong>{intf._status.ip4Addr.0}</strong>'
+                    }
+                }, {
+                    xtype: 'button',
+                    ui: 'action',
+                    text: 'Renew IP',
+                    handler: 'onRenew',
+                    bind: {
+                        disabled: '{intf.addressSource !== "dhcp" || intf.addressSource !== dhcpv6}'
```

Comment:
Is dhcpv6 a string? If so it requires quotes.
```suggestion
                        disabled: '{intf.addressSource !== "dhcp" || intf.addressSource !== "dhcpv6"}'
```